lib/legally.js

```diff
@@ -5,7 +5,12 @@ var open = require('./open');
 var folders = require('./folders');
 
 function getPackage(path){
-  var name = path.split(/\//).pop();
+  var name;
+  if (path.indexOf("@types") > 0) {
```

Comment:
Sorry, yeah, i can see this could need to be changed to >=0 for completeness

Since you had already merged this, I thought maybe you decided it was ok, hence #11.

I'll add a quick PR to change this one.